Sources/Sentry/SentrySDK.m

```diff
@@ -464,9 +464,19 @@ + (void)installIntegrations
     NSMutableArray<NSString *> *integrationNames =
         [SentrySDK.currentHub getClient].options.integrations.mutableCopy;
 
+    NSArray<Class> *defaultIntegrations = SentryOptions.defaultIntegrationClasses;
+    NSMutableDictionary<NSString *, Class> *integrationDictionary =
+        [[NSMutableDictionary alloc] init];
+
+    for (Class integrationClass in defaultIntegrations) {
+        integrationDictionary[NSStringFromClass(integrationClass)] = integrationClass;
```

Comment:
`h`: I don't get it. The code only writes to the dictionary but never reads from it. How does it solve the underlying problem?

Reply:
🤔
Strange, that is something missing here

Reply:
Super weird stuff, somehow I hade reverted the change I made.